### ISSUE TITLE
Option for removing MED policy from interface

### ISSUE THREAD

**Issue by maloy-ghosh** [vincentbernat/lldpd#292]
From the documents http://vincentbernat.github.io/lldpd/usage.html, it is possible to configure a MED policy via

```
configure [ports ethX [,…]] med policy application application [unknown] [tagged] [vlan vlan] [priority priority] [dscp dscp]
``` 
This adds Network Policy TLV to LLDP packets. But is there an option to remove (unconfigure) a policy from interface?

I was trying something like this in lldpcli (built from source on 7db1168 redhat: install lldpcli as setuid _lldpd)

```
[lldpcli] # unconfigure ports <name> med 
2018-07-12T14:25:38 [WARN/lldpctl] incomplete command
````

[lldpcli] # unconfigure ports <name> med <tab> does not show anything. 

Am I missing anything? What is the correct syntax of the unconfigure port med commands?

**Comment by vincentbernat** [comment]
This hasn't been implemented. The manual page is extensive and if you don't have an `unconfigure` option specified, it doesn't work. You can use `unconfigure med fast-start` but it's not available as a per-port setting. This explains why you can go to `med` and not have completion after that.

I think unconfiguring a MED policy should not be very complex to do.

**Comment by maloy-ghosh** [comment]
So locally I have fixed the issue by creating a command `configure ports c_1_7 med policy application <app> disabled` and handled it according to other fields (like unknown/tagged etc)

Would such a change be accepted upstream? Then I can prepare a formal pull request for it.

**Comment by vincentbernat** [comment]
Yes, please, open a PR. It should be pretty simple to change your patch to have `unconfigure ports <port> med policy application <app>` instead. If you can't figure out, open a PR either way, I'll fix that.
